src/components/layout/MainNavigation.js

```diff
@@ -41,6 +50,14 @@ function MainNavigation({setRefreshPost}) {
       ) : (
         ""
       )}
+      {showError ? (
```

Comment:
Can this be done using ?? syntax or does that cause errors?

Reply:
it probably could, and that would remove a lot of unnecessary lines